docs-src/cloud/cloud-intro.md

```diff
@@ -14,6 +14,7 @@ To request a Temporal Cloud account, complete the [request form](https://pages.t
 :::
 
 - [Introduction to Temporal Cloud](/cloud/introduction)
+  - [Security](/cloud/security)
```

Comment:
link to the node id of the first node in the guide - and link magic will do this 
Plus we can be sure the links are rewritten for guide indexing - so urls in result matches actually work.